src/Microsoft.Data.SqlClient/netcore/ref/Microsoft.Data.SqlClient.cs

```diff
@@ -4,6 +4,10 @@
 
 // NOTE: The current Microsoft.VSDesigner editor attributes are implemented for System.Data.SqlClient, and are not publicly available.
 // New attributes that are designed to work with Microsoft.Data.SqlClient and are publicly documented should be included in future.
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
```

Comment:
NIT, following other declarations, uses a full name instead of adding using keyword.